test/sql/insert.sql.in

```diff
@@ -118,3 +118,33 @@ EXPLAIN (costs off) INSERT INTO date_inf SELECT * FROM date_inf
     WHERE time <= '-infinity' LIMIT 1;
 EXPLAIN (costs off) INSERT INTO date_inf SELECT * FROM date_inf
     WHERE time > '-infinity' LIMIT 1;
+
+-- test INSERT with cached plans / plpgsql functions
+-- https://github.com/timescale/timescaledb/issues/1809
+CREATE TABLE status_table(a int, b int, last_ts timestamptz, UNIQUE(a,b));
+CREATE TABLE metrics(time timestamptz NOT NULL, value float);
+CREATE TABLE metrics2(time timestamptz NOT NULL, value float);
+SELECT (create_hypertable(t,'time')).table_name FROM (VALUES ('metrics'),('metrics2')) v(t);
+
+INSERT INTO metrics VALUES ('2000-01-01',random()), ('2000-02-01',random()), ('2000-03-01',random());
+
+CREATE OR REPLACE FUNCTION insert_test() RETURNS VOID LANGUAGE plpgsql AS $$
```

Comment:
Nit: would be nice to have the start `$$` marker at the same level/indentation as the end one.